doc/source/ray-core/patterns/limit-pending-tasks.rst

```diff
@@ -0,0 +1,45 @@
+Pattern: Using ray.wait to limit the number of pending tasks
```

Comment:
can we add a sphinx target here, so that we can reference it instead of the file?

btw. that is the recommended way of doing it as the documentation structure may change, leading to broken links, while targets are valid no matter where the file is moved to

Reply:
It's actually something I don't fully understand: if the file is moved, broken links will be caught by our CI and they can be fixed easily. By referencing the doc file, sphinx will add the doc title which is kind of nice...

But I can add a sphinx target :)

Reply:
IMO the important thing is not to use absolute path because we want to reference to other parts in the *same* version of release.
Other than that, I am not sure if one way is preferred that the other.

Reply:
Yea, both approaches will guarantee that since we don't hard code "http://docs.ray.io...."